newton/_src/utils/add_bdx_physics_schema.py

```
@@ -0,0 +1,390 @@
+    rods = [
+        ("HangingLanternChainA_01", "HangingLanternC_01"),  # 3
+        ("HangingLanternChainA_02", "HangingLanternE_01"),  # 2
+        ("HangingLanternChainA_03", "HangingLanternD_01"),  # 1
+        ("HangingLanternChainA_04", "HangingLanternA_03"),  # 1
+        ("HangingLanternChainA_05", "HangingLanternA_01"),  # 1
+        ("HangingLanternChainA_06", "HangingLanternA_02"),  # 2
+        ("HangingLanternChainA_07", "HangingLanternC_02"),  # 2
+    ]
+
+    joints = []
+    world = stage.GetPrimAtPath("/World")
```

Ruff (0.14.8)

Local variable `world` is assigned to but never used

Remove assignment to unused variable `world`

(F841)